### ISSUE TITLE
Multiplayer Cant see other players

### ISSUE THREAD

**Issue by Urammar** [shinyquagsire23/OpenJKDF2#246]
Hi, playing with my bro, whoever joins last cant see the other players, its very inconstant.

I can replicate the same problem running multiple clients locally. One player can see all the others, but the latest joiner cannot see anyone else.

In addition, in co-op, the people and meshes are very strange for a client

**Comment by Pheener** [comment]
I ran into this too, while testing it out with a friend. As far as I can tell: 

- The first player to connect (either as the server, or as the first person connecting to the dedicated server) cannot see any clients visually. 
- The dedicated server can see both clients visually.
- The client can see the host visually, but not vice versa.  
- Sounds (footsteps, weapons) and weapon fire seems to be synced.

I tested this both with my friend, remotely, **and** with 3 instances on the same computer, connecting to the same game, using the same player model (just in case!)

Hope that helps, and thanks for all your hard work!

**Comment by Urammar** [comment]
Yup can confirm thats my exact issue also. Should have said that, sound effects (walking, weapons, jumping) all replicated correctly. You can even shoot eachother. But the actual player model is not visible.

I wonder if its a loading bug, like they arent technically in the world so they werent loaded or something? But again even locally with multiple game instances, the behavior is the same

**Comment by Urammar** [comment]
Hey just throwing some more attention on this. It's not just me, and it makes me sad ):

**Comment by DonSchlitterbahn** [comment]
I had the same issue so I went back quite a few versions and didn't have the issue anymore. I think it was v0.7.0 64-bit Debug, but I definitely might be wrong because it's been months since I last played. This problem spans pretty much every version since. The issue with v0.7.0 64-bit Debug was that some areas of levels would crash 100% of the time for the client which ultimately ended the playthrough. I play using a mobile hotspot so perhaps that could be a thing as well since you mentioned high ping.

**Comment by shinyquagsire23** [comment]
ok so, I made a mistake and mistook this issue for another one that I had before, where not seeing other players was just intermittent and not 100% of the time. Turns out I broke multiplayer on DF2 only with my MoTS release and it was just sitting broken for a while now

**Comment by shinyquagsire23** [comment]
Should be fixed (at least to where it was before, which was basically working 80% of the time) with v0.8.19

**Comment by Urammar** [comment]
Okay seems to be.

Be advised, in capture the flag modes, only one player is actually spawned into the mode, all other players do actually spawn in the match but dont get a colour nor can the interact with the flag robot things.

They are spawned at respawn points as deathmatch players or whatever I guess?

Unsure if this should be its own bug or stay here, but the initial problem is fixed, and most multiplayer is at least possible now. So well done and congrats, and also big thank you!!!

**Comment by Urammar** [comment]
@shinyquagsire23 I think you missed this after adding the fixed label. Havent tested with the new version yet, though.
